### ISSUE TITLE
Support 'notes' field on create/update operations

### ISSUE THREAD

**Issue by maniax89** [IBM-Bluemix/gp-js-client#54]
According to the swagger spec, the field `notes` is supported on get/create/update for a few places:

- [x] **bundle get**: https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/getBundleInfo
- [x] **bundle create**: https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/createBundle
- [x] **bundle update**: https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/updateBundle
- [ ] **resource entries get** (not explicit, assuming part of `resourceEntries`): https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/getResourceStrings
- [ ] **resource entries create** (not explicit, assuming part of `resourceEntries`): https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/uploadResourceStrings
- [ ] **resource entries update** (not explicit, assuming part of `resourceEntries`): https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/updateResourceStrings
- [x] **resource entry get**: https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/getResourceEntryInfo
- [x] **resource entry update**: https://gp-rest.ng.bluemix.net/translate/swagger/index.html#!/bundle/updateResourceEntryInfo

Would be great if we could have access to this `notes` field across these endpoints

**Comment by srl295** [comment]
@yumaoka are notes part of `bundle/getResourceStrings` / `bundle/uploadResourceStrings` / `bundle/updateResourceStrings` ?

**Comment by srl295** [comment]
@maniax89 i opened a PR above ^ that supports all of the 'explicit' modes of notes fields.

**Comment by yumaoka** [comment]
@srl295 

>@yumaoka are notes part of bundle/getResourceStrings / bundle/uploadResourceStrings / bundle/updateResourceStrings ?

Yes.

Also, resource entry APIs: getResourceEntryInfo / updateResourceEntryInfo.
And, bundle APIs (for bundle wide notes) : getBundleInfo / createBundle / updateBundle.

**Comment by srl295** [comment]
@yumaoka 
> Yes

Thanks, I will look into this and add them.

> Also, resource entry APIs: getResourceEntryInfo / updateResourceEntryInfo. And, bundle APIs (for bundle wide notes) : getBundleInfo / createBundle / updateBundle

Got that… already supported in this PR.

**Comment by srl295** [comment]
(for reference)

for `bundle/uploadResourceStrings` this `body:` payload works:

```json
{
  "key": {
      "value": "value",
     "notes": [ "abc","def"]
   }
}
```

**Comment by srl295** [comment]
Landed above support in c2893abf28c6aea7f2f1fad03ad0be81aabac6b6, will work on the bulk operations in #65 (next release)
